apis/control/v1alpha1/cassandratask_types.go

```diff
@@ -81,6 +81,7 @@ const (
 	CommandGarbageCollect  CassandraCommand = "garbagecollect"
 	CommandFlush           CassandraCommand = "flush"
 	CommandRefresh         CassandraCommand = "refresh"
+	CommandInodeTSReload   CassandraCommand = "inodetsreload"
```

Comment:
issue: Wasn't this tsreload? inode means nothing in cass-operator, we always use "internode" in its full form and in some cases (metrics) the actual "inode" which means something quite different. I'm not sure why we would try to save few characters here. Rather, just remove "inode" entirely if needed and we can later extend this feature to include client / server selection through args.

Reply:
Done.